docs/docs/usage.md

```diff
@@ -0,0 +1,122 @@
+# Usage
+
+## Publish to RStudio Connect
+
+To publish to RStudio Connect:
+
+- Open a Jupyter notebook.
+- Click the <img class="icon"; src="../images/publish-icon.gif" alt="blue toolbar icon used for publishing the notebook"> icon (blue publish button) and select **Publish to RStudio Connect**
+to publish the current notebook to RStudio Connect.
+
+!!! note
+    This plugin is only for notebooks using Python kernels. Therefore, R notebooks cannot be published using this plugin.
+
+## Entering server information
+
+- If this is your first time publishing a notebook, you will be
+prompted to enter the location and a nickname for the RStudio Connect server.
+- You will also be prompted to enter your API Key. See the [RStudio Connect User
+Guide](http://docs.rstudio.com/connect/user/api-keys) for
+instructions on generating API Keys for your user.
+- When you click the **Add Server** button, `rsconnect-jupyter` will send a request to the RStudio Connect server to verify that it can be reached via the requested URL and that the API key is valid.
+
+If your RStudio Connect server was configured with a self-signed certificate (or other certificates that the computer hosting your Jupyter notebook server does not trust), the attempt to contact RStudio Connect may fail with a TLS-related error.
+
+You have multiple options in this case, depending on your needs:
+
+1. If your RStudio Connect Administrator can give you the Certificate Authority (CA)
```

Comment:
Do the administrator references here need to be product-specific?

Reply:
Change to "administrator"?